scripts/run_on_kube.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/python
```

Comment:
@jkinkead this script creates a kubernetes configuration that repeatedly fails (due to `touch: cannot touch '/net/efs/aristo/helloworld': Read-only file system`).  I would prefer that it fails once, and does not try again.  But instead it repeatedly spins up pods, seemingly bogging down the kubernetes server.  I specified `spec.completions: 1` but that seemed to have no effect.  Any ideas on how I can stop this behavior?